File: WebContent/clientMVC/scripts/model/loginSO.js
var LoginSO = function() {
	this.loginDAO = new LoginDAO();
};
	
LoginSO.prototype.confirmLogin = function(loginDO) {
	var result = false;
	
	if(loginDO.getId() != null && loginDO.getPasswd() != null) {
		result = this.loginDAO.checkLogin(loginDO);
	}
	
	return result;
};

LoginSO.prototype.confirmLogout = function() {
	var result = false;
	
	if(this.loginDAO.logout()) {
		result = true;
	}
	
	return result;
};

File: src/modelForServerMVC/LoginSo.java
package modelForServerMVC;

public class LoginSo {

	private LoginDao loginDao;
	
	public LoginSo() {
		loginDao = new LoginDao();
	}
	
	public boolean confirmLogin(LoginDo loginDo) {
		return null != loginDo.getId() && null != loginDo.getPasswd() && loginDao.checkLogin(loginDo);
	}
}
